project_sequence/tests/test_something.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2023 Moduon Team S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0)
+
+from freezegun import freeze_time
+
+from odoo.tests.common import Form, TransactionCase, new_test_user, users
+
+
+@freeze_time("2023-01-01 12:00:00", tick=True)
+class SomethingCase(TransactionCase):
```

Comment:
Here too, "Something" is just a placeholder. Give it a meaningful name:

```suggestion
class ProjectSequenceCase(TransactionCase):
```